client/src/pages/Dashboard/MediaContainer.jsx

```diff
@@ -3,7 +3,7 @@ import MediaRenderer from "./MediaRenderer";
 import dashboardStyles from "./DashboardPage.module.css";
 
 const MediaContainer = ({ mediaUrls }) => {
-  const numberOfMedia = mediaUrls.length;
+  const numberOfMedia = mediaUrls?.length;
```

Comment:
fyi: https://www.freecodecamp.org/news/how-the-question-mark-works-in-javascript/

Reply:
This solution results with a different UI bug: 
<img width="1107" alt="image" src="https://github.com/hsu-isaac/fitconnect/assets/85271794/1f8b2dc5-cb4a-4d99-ac8d-8fff97c50457">

I have a solution in the branch that I'm working on now that handles this without rendering a large blank space for the media container, would you be ok with reverting this fix and pulling my branch into this after I push it to main and use [that solution](https://github.com/hsu-isaac/fitconnect/blob/add-edit-functionality/client/src/pages/Dashboard/Dashboard.jsx#L46C1-L49C19) instead?
It's essentially the same solution, I just have it one level higher for conditional container rendering

Reply:
yeah absolutely!! haha I wasn't sure what I was supposed to see/not see, I thought that was what the medias looked like since I was able to see some other medias! Let's ignore this and merge your fixes instead